data/json/items/tools.json

```diff
@@ -6256,6 +6286,21 @@
     "color": "brown",
     "qualities": [ [ "DIG", 2 ], [ "COOK", 1 ], [ "BUTCHER", -96 ] ]
   },
+  {
+    "id": "makeshift_shovel",
```

Comment:
I need to revisit what DIG 2 implies, but I'm extremely skeptical of lashing a piece of sheet metal (or any kind of stone implement?) to a stick and calling it a shovel.

Reply:
I think a generic shovel IS a piece of (press-formed) sheet metal attached to a (polished) stick.

Reply:
[primitive stone shovel](https://www.youtube.com/watch?v=hy7-dbHSFNA) if the stone works the metal should too, it does seem like the rock kind moved a bit so maybe it wouldn't last forever, this could be simulated by having the tool degrade and need maintenance, but I don't know if that can be done.

Reply:
> 
> 
> I think a generic shovel IS a piece of (press-formed) sheet metal attached to a (polished) stick.

Well, that sheet metal has to be very solid to withstand torque while digging and not to bend. I've tried to cut a piece of shovel blade with a hacksaw once, and the shovel won the fight: hacksaw lost a lot of teeth and cutting progress was extremely slow. Stone actually can be more solid than random sheet of soft steel... unless you use piece of stell with the similar shape as a proper stone...